docs/mkdocs.yml

```diff
@@ -0,0 +1,155 @@
+site_name: Hyperledger Caliper
+repo_name: hyperledger/caliper
+repo_url: https://github.com/hyperledger/caliper
+docs_dir: source
+site_dir: build
+theme:
+  name: material
+  custom_dir: overrides
+  logo: _static/caliper.svg
+  favicon: _static/caliper.svg
+  icon:
+    repo: fontawesome/brands/github
+  features:
+    - announce.dismiss
+    - content.code.annotate
+    - content.code.copy
+    # - content.code.select
+    # - content.footnote.tooltips
+    - content.tabs.link
+    - content.tooltips
+    # - header.autohide
+    - navigation.footer
+    - navigation.indexes
+    # - navigation.instant
+    # - navigation.instant.prefetch
+    # - navigation.instant.progress
+    # - navigation.prune
+    - navigation.sections
+    - navigation.tabs
+    - navigation.tabs.sticky
+    - navigation.top
+    - navigation.tracking
+    - search.highlight
+    - search.share
+    - search.suggest
+    - toc.follow
+    # - toc.integrate
+  palette:
+  # Palette toggle for light mode
+  - media: "(prefers-color-scheme: light)"
+    scheme: default
+    toggle:
+      icon: material/brightness-7
+      name: Switch to dark mode
+  - media: "(prefers-color-scheme: dark)"
+    # Palette toggle for dark mode
+    scheme: slate
+    toggle:
+      icon: material/brightness-4
+      name: Switch to light mode
+  font:
+    text: Roboto
+    code: Roboto Mono
+  favicon: assets/favicon.png
+  icon:

```

Comment:
The properties `favicon` and `icon` are already defined above; the editor gives a duplication error. Mkdocs probably uses the first occurrences, but we shouldn't leave that tp chance, so please only set the attributes once.